tests/conftest.py

```diff
@@ -178,3 +178,20 @@ def multi_threading_test_devices():
         "nidaqmx\\tests\\max_config\\nidaqmxMaxConfig.ini to create these devices."
     )
     return None
+
+
+@pytest.fixture(scope="module")
+def test_device(request):
```

Comment:
Prefixing the fixture with `test_` does not make a meaningful distinction because all test fixtures are for tests.

Suggestion: something like `named_device` or `device_by_name`

Reply:
updated the name of the fixture to `device_by_name`